deps/verifier/src/sample/mod.rs

```diff
@@ -14,6 +14,20 @@ struct SampleTeeEvidence {
     report_data: String,
 }
 
+#[derive(Deserialize, Debug)]
+pub struct Evidence {
+    pub svn: String,
+    pub report_data: String,
+    pub tpm_quote: TpmQuote,
```

Comment:
I would probably keep the sample verifier and the TPM verifier totally separate, but I'm not quite sure what you have in mind.

Reply:
My thinking was using tpm quote in sample attester for testing. But re-thinking about it, it's not really needed as we can use swtpm. I'll remove it

Reply:
This will also avoid changing the GHA which are failing due to missing TPM deps